packages/sku/src/services/vite/plugins/middlewarePlugin.ts

```diff
@@ -93,7 +96,7 @@ export const middlewarePlugin = (skuContext: SkuContext): Plugin => ({
         try {
           const { viteRender } = await server.ssrLoadModule(renderEntry);
 
-          const renderedHtml = await (viteRender as ViteRenderFunction)({
+          let html = await (viteRender as ViteRenderFunction)({
```

Comment:
Made these changes when experimenting with `beasties`, and decided to keep them. Having a single `html` variable that's incrementally modified makes it much easier to transform the HTML over separate steps, rather than dealing with many different named html constants while also ensuring you refer to the correct one in the next transformation.